src/components/InvitePopover.tsx

```diff
@@ -77,7 +77,7 @@ export function InvitePopover(props: Props): React.ReactElement {
   return (
     <>
       <IconButton onClick={handleClick}>
-        <PersonAdd />
+        <Edit />
```

Comment:
@joshuajames-smith is there a custom icon we can use here for consistency with other icons?

Reply:
Done